lib/modules.nix

```diff
@@ -10,26 +10,20 @@ with lib.types;
 
 rec {
 
-  /* Evaluate a set of modules.  The result is a set of two
+  evalModules = import ./eval-modules.nix;
+
+  /* Evaluate a set of modules. The result is a set of two
      attributes: ‘options’: the nested set of all option declarations,
-     and ‘config’: the nested set of all option values.
-     !!! Please think twice before adding to this argument list! The more
```

Comment:
This comment should probably be retained and put above the new `evalModules` and maybe indicate that they are there only for backwards compatibility, and as such should be considered deprecated.